BrainPortal/app/controllers/nh_invitations_controller.rb

```diff
@@ -40,12 +40,20 @@ def create #:nodoc:
     user_emails     = (params[:emails].presence.try(:strip) || "").split(/[\s,]+/)
     user_emails     = user_emails.map(&:presence).compact
     cb_error "Please specify at least one email address", :redirect => nh_project_path(@nh_project) if user_emails.empty?
-    user_ids        = User.where(:email => user_emails).pluck(:id)
+    users           = User.where(:email => user_emails)
+    user_ids        = users.pluck(:id)
+    found_emails    = users.pluck(:email)
+    wrong_emails    = user_emails - found_emails
+
+    flash_messages = []
+    if wrong_emails.present?
+      flash_messages << "\nWe are not able to invite user(s) with email(s) #{wrong_emails.join(", ")}. At the moment users are matched by email that they are used to register in NeuroHub. Please confirm with them which email they provided to NeuroHub. "
```

Comment:
This PR is fine, I just would like to see some more '\n' in the message, it give a really long message that becomes not readable. 

The message regarding the failing invitation is actually in green (notice) I would like prefer to have it in red (error).

Reply:
I would call it pink but resolved. I also added yellow color flash, hope it is fine @prioux @candicecz ?